tests/golden_tests/mutual_recursion/multiple.hvm

```diff
@@ -0,0 +1,11 @@
+(A) = (B)
+(B) = (C)
+(C) = (A)
+
+(H) = (I)
+(I) = (H)
+
+(M) = (N)
```

Comment:
Can you change this to have a cycle of length one? Like `(M) = (M)`. Might need to be something more complex not to get an error earlier, like `N = @x (x (N x))`